### PR TITLE
Fix 3383 kw wizard classified raster

### LINKED ISSUE #3383: Wizard for Classified Raster Layer
Currently, the wizard works classified vector layer. We will need to make it work also for classified raster layer.

To do:
- [x] Update how to get the class value from cell, not only from field
- [x] Update the class mapping step work with raster layer
- [x] Add unit test for classified raster data

### PR DESCRIPTION
**Pull request by ismailsunni**
Fix #3383

### CODE REVIEW

File: safe/test/utilities.py [hunk 1]
```diff
@@ -426,6 +426,7 @@ def load_test_raster_layer(*args, **kwargs):
     .. versionadded:: 4.0
     """
     path = standard_data_path(*args)
+    assert os.path.exists(path), 'Path %s is not exist' % path
```
Comment (Gustry) [inline safe/test/utilities.py]: I'm not a pro in english, but `Path do not exist` or `Path is not existing` ;-)

Comment (Gustry) [inline safe/test/utilities.py]: I didn't know we could assert outside of a test

Comment (ismailsunni) [inline safe/test/utilities.py]: Ah yes, you are right about the English. Sometimes my English is off when there is '%s'. :D Thanks.

Well, we can. But, the main point of this to see in travis which path is not exist.

File: safe/gui/tools/wizard/test/test_keyword_wizard.py [hunk 2]
```diff
@@ -1185,6 +1189,278 @@ def test_existing_exposure_population_polygon_keyword(self):
 
         self.assertDictEqual(real_keywords, expected_keyword)
 
+    # @unittest.skip('Path is not found')
```
Comment (Gustry) [inline safe/gui/tools/wizard/test/test_keyword_wizard.py]: Do you need it ?

**Pull request merged**